Dojo_class.py

```diff
@@ -0,0 +1,121 @@
+import random
+from Room_Class import *
+from Person_Class import *
+
+
+class Dojo(object):
+    def __init__(self):
+        self.all_rooms = []
```

Comment:
The all_rooms variable can store room attributes. This should eliminate the need for all_offices and all_living_spaces